src/query/queryWithCursor.ts

```diff
@@ -3,7 +3,7 @@ import { decrypt, encrypt } from '../utils';
 import { AnyObject, Filter, TableConfig } from '../types';
 import { buildQueryTableParams } from './queryBuilder';
 
-const MAX_LIMIT = 50;
+const DEFAULT_LIMIT = 99999;
```

Comment:
Why such a big number `99999` ? 

Should loading `100` records by default not be sufficient ? User already has property to override it if they want.

Reply:
No, in progress orders may contain any count and sure it will be less than `99999`